doc/management/private.md

```diff
@@ -1,9 +1,178 @@
 # Private API for Management
 
+## Enum `Status`
```

Comment:
Please try to document the same type only once at a single place, maybe on the overview page. Also "StaffStatus" would be good as type.

Reply:
I agree, having a global enum should only be defined once.